src/cf.data.pre

```diff
@@ -473,6 +473,26 @@ DOC_START
 	CAP_IPC_LOCK capability, or equivalent.
 DOC_END
 
+NAME: hopeless_kid_revival_delay
+COMMENT: time-units
+TYPE: time_t
+LOC: Config.hopelessKidRevivalDelay
+DEFAULT: 60 minutes
```

Comment:
Since `60 minute` is grammatically incorrect, the current parsing code supports `60 minutes`, and many existing defaults already use `minutes`, I think `60 minutes` is acceptable. However, please use `1 hour` instead to avoid debating a very minor issue.